internal/application/filesystem.go

```diff
@@ -40,19 +40,22 @@ import (
 	log "github.com/sirupsen/logrus"
 
 	"github.com/konveyor/move2kube-api/internal/types"
+	"github.com/otiai10/copy"
 )
 
 const (
 	assetsDirectory         = string(ApplicationStatusAssets)
 	artifactsDirectoryName  = string(ApplicationStatusArtifacts)
 	archivesDirectory       = "archives"
 	srcDirectory            = "src"
+	containersDirectory     = "containers"
 	m2kplanfilename         = appNameShort + ".plan"
 	m2kQAServerMetadataFile = "." + appNameShort + "qa"
 	m2kPlanOngoingFile      = "." + appNameShort + "plan"
 	apiServerPort           = 8080
 	timestampRegex          = `time="\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z"`
 	m2kString               = "Move2Kube"
+	additionalFilename      = "m2kadditionalfiles.txt"
```

Comment:
Can we call it say newfiles.txt? Also, rename the variable name to something like say `newfilesMetadataFileName` or something similar.

Reply:
Yes sure.